indigo_api/migrations/0128_rename_badges.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-03-26 14:02
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Group
+from django.db import migrations
+from pinax.badges.models import BadgeAward
+
+
+def rename_badges(apps, schema_editor):
+    """ Rename badges, and delete old security groups
+    """
+    slugs = [
+        ('drafter', 'editor'),
+        ('senior-drafter', 'reviewer'),
+    ]
+    security_groups = ['Drafter Badge', 'Senior Drafter Badge']
+
+    for old, new in slugs:
+        BadgeAward.objects.filter(slug=old).update(slug=new)
+
+    for g in security_groups:
+        try:
+            group = Group.objects.get(name=g)
+            group.delete()
+        except Group.DoesNotExist:
+            pass
+
+
+def unrename_badges(apps, schema_editor):
+    """ Rename badges, and delete new security groups
+    """
+    slugs = [
```

Comment:
probably better to make `slugs` a global, but up to you. reduces changes of typos.